client/src/cards/card-plate-appearance.jsx

```diff
@@ -1147,10 +1148,14 @@ class CardPlateAppearance extends React.Component {
   }
 
   renderWalkupSong() {
+    const noSongClickHandler = () => {
+      setRoute(`/players/${this.props.player.id}/edit`);
+    };
```

Comment:
To improve perf, this could be defined outside the render method so a new function doesn't get created on each render.